tools/gulp/tasks/development.ts

```diff
@@ -143,7 +143,8 @@ task(':watch:devapp', () => {
   // Material package watchers.
   watchFilesAndReload([
     join(materialPackage.sourceDir, '**/!(*-theme.scss)'), `!${materialCoreThemingGlob}`
-  ], ['material:build-no-bundles']);
+  // Note that we need to rebuild the dev app CSS, because the typography may have changed.
+  ], ['material:build-no-bundles', ':build:devapp:scss']);
```

Comment:
Can the typography also change for JS files?

Reply:
No it shouldn't. I got it working a bit cleaner by updating the glob instead.

Reply:
Yeah, that now looks good.